### PR TITLE
Follow english documents

### PR DESCRIPTION
**Pull request by LittleAaron**
Add description for AppArmor.
Update AppArmor requirement section.

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
Please sign CLA
https://github.com/kubernetes/website/pull/30701#issuecomment-983475678

File: content/zh/docs/tutorials/clusters/apparmor.md [hunk 1]
```diff
@@ -29,6 +29,14 @@ AppArmor 是通过配置文件进行配置的，这些配置文件被调整为
 每个配置文件都可以在*强制（enforcing）*模式（阻止访问不允许的资源）或*投诉（complain）*模式
 （仅报告冲突）下运行。
 
+<!-- AppArmor can help you to run a more secure deployment by restricting what containers are allowed to
+do, and/or provide better auditing through system logs. However, it is important to keep in mind
+that AppArmor is not a silver bullet and can only do so much to protect against exploits in your
+application code. It is important to provide good, restrictive profiles, and harden your
+applications and cluster from other angles as well. -->
+AppArmor 可以通过限制允许容器执行的操作来帮助您运行更安全的部署，和/或通过系统日志提供更好的审计。
```
Comment (tengqm) [inline content/zh/docs/tutorials/clusters/apparmor.md:37]:
```suggestion
AppArmor 可以通过限制容器可以执行的操作和/或通过系统日志提供更好的审计来帮助你运行更安全的部署。 
```

File: content/zh/docs/tutorials/clusters/apparmor.md [hunk 2]
```diff
@@ -29,6 +29,14 @@ AppArmor 是通过配置文件进行配置的，这些配置文件被调整为
 每个配置文件都可以在*强制（enforcing）*模式（阻止访问不允许的资源）或*投诉（complain）*模式
 （仅报告冲突）下运行。
 
+<!-- AppArmor can help you to run a more secure deployment by restricting what containers are allowed to
+do, and/or provide better auditing through system logs. However, it is important to keep in mind
+that AppArmor is not a silver bullet and can only do so much to protect against exploits in your
+application code. It is important to provide good, restrictive profiles, and harden your
+applications and cluster from other angles as well. -->
+AppArmor 可以通过限制允许容器执行的操作来帮助您运行更安全的部署，和/或通过系统日志提供更好的审计。 
+但是，重要的是要记住 AppArmor 不是灵丹妙药，只能做很多事情来防止应用程序代码中的漏洞利用。
```
Comment (tengqm) [inline content/zh/docs/tutorials/clusters/apparmor.md:38]:
```suggestion
但是，重要的是要记住 AppArmor 不是灵丹妙药，只能做有限的工作来防止应用程序代码中的漏洞利用。 
```

File: content/zh/docs/tutorials/clusters/apparmor.md [hunk 3]
```diff
@@ -29,6 +29,14 @@ AppArmor 是通过配置文件进行配置的，这些配置文件被调整为
...
 （仅报告冲突）下运行。
 
+<!-- AppArmor can help you to run a more secure deployment by restricting what containers are allowed to
+do, and/or provide better auditing through system logs. However, it is important to keep in mind
+that AppArmor is not a silver bullet and can only do so much to protect against exploits in your
+application code. It is important to provide good, restrictive profiles, and harden your
+applications and cluster from other angles as well. -->
+AppArmor 可以通过限制允许容器执行的操作来帮助您运行更安全的部署，和/或通过系统日志提供更好的审计。 
+但是，重要的是要记住 AppArmor 不是灵丹妙药，只能做很多事情来防止应用程序代码中的漏洞利用。 
+重要的是提供良好的、限制性的配置文件，并从其他角度强化您的应用程序和集群。
```
Comment (tengqm) [inline content/zh/docs/tutorials/clusters/apparmor.md:39]:
```suggestion
重要的是提供良好的、限制性较强的配置文件，并从其他角度加固你的应用程序和集群。
```

File: content/zh/docs/tutorials/clusters/apparmor.md [hunk 4]
```diff
@@ -320,7 +309,7 @@ kubectl get events | grep hello-apparmor
 我们可以通过检查该配置文件的 proc attr 来验证容器是否实际使用该配置文件运行：
 
 ```shell
-kubectl exec hello-apparmor -- cat /proc/1/attr/current
+kubectl exec hello-apparmor cat /proc/1/attr/current
```
Comment (tengqm) [inline content/zh/docs/tutorials/clusters/apparmor.md:312]: 英文中是有 `--` 的

File: content/zh/docs/tutorials/clusters/apparmor.md [hunk 5]
```diff
@@ -330,7 +319,7 @@ k8s-apparmor-example-deny-write (enforce)
 最后，我们可以看到如果试图通过写入文件来违反配置文件，会发生什么情况：
 
 ```shell
-kubectl exec hello-apparmor -- touch /tmp/test
+kubectl exec hello-apparmor touch /tmp/test
```
Comment (tengqm) [inline content/zh/docs/tutorials/clusters/apparmor.md:322]: 同上

**Comment by jimangel** [comment]
Hey there! @LittleAaron, looks like you haven't signed the CLA yet. Could I please have you do that? https://github.com/kubernetes/community/blob/master/CLA.md

**Comment by avinashupadhya99** [comment]
Hi @LittleAaron! Just a friendly reminder to sign the [CLA](https://github.com/kubernetes/community/blob/master/CLA.md) for the PR to be further reviewed.

**Comment by tengqm** [comment]
CLA is not okay and the author is not responding for over a month.
/close

**Pull request closed without merging**